dbld/images/ubuntu_1404_trusty.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, Balazs Scheidler <balazs.scheidler@oneidentity.com>"
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN=true
+ENV LANG C.UTF-8
+ENV OS_PLATFORM trusty
+
```

Comment:
why does it not have to match the distro names in the packages.txt file?

Ah, I think I get it now, it does not need to match as long as "trusty" as a string is present. I think this is not a good idea, and we should use an exact match.

also note that OS_PLATFORM as an environment variable is exported to dependencies.sh, so no need to pass it as an argument, it should be available as an environment variable, making the dockerfile a bit more concise.